app/models/phone_number.rb

```diff
@@ -0,0 +1,5 @@
+class PhoneNumber < ApplicationRecord
+  PHONE_TYPE = %w[mobile landline].freeze
+  validates_presence_of :number, :created_at, :updated_at
+  has_and_belongs_to_many :patients
```

Comment:
This should be a `has_many :through` relationship, as we may want to flag phone numbers as inactive for a certain patient.

Also, HABTM relationships get messy quickly. I strongly prefer having an explicit join table.